### ISSUE TITLE
Throw better error for invalid code

### ISSUE THREAD

**Issue by fisker** [postcss/postcss-scss#110]
```js
require("postcss-scss").parse('@import ur l(//a)')
```

Throws 

```text
Uncaught TypeError: Cannot read property 'line' of null

      at ScssParser.atrule (node_modules/postcss-scss/node_modules/postcss/lib/parser.js:362:21)
      at ScssParser.parse (node_modules/postcss-scss/node_modules/postcss/lib/parser.js:51:16)
      at scssParse (node_modules/postcss-scss/lib/scss-parse.js:9:10)
     
```

Should throw a better error or something else.

**Comment by ai** [comment]
Thanks. I will fix it tomorrow.

Can you show the full stack trace to see line numbers?

**Comment by fisker** [comment]
Sure, will update later.

**Comment by ai** [comment]
Fixed https://github.com/postcss/postcss/commit/2f38bebaa206bd72b3d5f9ba5ae34ae5a3c1b155

**Comment by ai** [comment]
The fix was released in 8.0.4
